pages/tidal tools/analyze_database.md

```diff
@@ -61,36 +61,45 @@ That is all, You should be able to see the results in your workspace within seco
 
 ## Create your Database configuration file
 
-Your YAML configuration file must contain the following information.
+For the most part, all supported Database engines share very similar configuration with Oracle being the exception.
+
+Your YAML configuration file for Postgres, MySQL, and SQL Server must contain the following information.
 
   - `id` - The id of the database from your Tidal Migrations account. You can find it in the URL bar when looking at a database instance. ex. If you are viewing a database instance in Tidal Migrations, the URL will show https://demo2.tidalmg.com/#/database_instances/111 in this case 111 is the database instance ID.
-  - `engine` - The database vendor, either `Oracle`, `SQL Server`, `MySQL`, or `PostgreSQL`, it is not case sensitive.
+  - `name` - A common name for your database could be the same or different from db_name, but this value is arbitrary and only for your reference.
+  - `engine` - The database vendor, either `SQL Server`, `MySQL`, or `PostgreSQL`, it is not case sensitive.
   - `host` - The hostname of the server that the database is located on and is accessible via a network connection from your current device and location.
-  - `port` - The port that the host has open and the database can accept connections on, the default for Oracle is `1521`, for SQL Server it is `1433`, for MySQL it is `3306`, and for PostgreSQL the default port is `5432`.
-  - `db_name` - The name of the database that will be analyzed, as it is defined within the database engine itself. ie. the value that is used by applications to connect to the database by name.
+  - `port` - The port that the host has open and the database can accept connections on, for SQL Server it is `1433`, for MySQL it is `3306`, and for PostgreSQL the default port is `5432`.
+  - `db_name` - The name of the database you are analyzing // or in the case of Oracle, the "service name". 
   - `user` - A username to authenticate with the database, check the section below for more details about creating a user and granting permissions.
   - `password` - A password for the corresponding user.
-  - `name` - A common name for your database could be the same or different from db_name, but this value is arbitrary and only for your reference.
 
 
-The simplest way to create your configuration file is to use the  `tidal analyze db init` command in Tidal Tools. It is an interactive command that will guide you through the process. Or you can create the file manually:
+When analyzing Oracle databases, the attributes in your configuration file must contain the following details.
 
-This is an example of how the YAML file should look like
+  - `id` - The id of the database from your Tidal Migrations account. You can find it in the URL bar when looking at a database instance. ex. If you are viewing a database instance in Tidal Migrations, the URL will show https://demo2.tidalmg.com/#/database_instances/111 in this case 111 is the database instance ID.
```

Comment:
Nice, super helpful to someone who might think `id` is the database ID and not a Tidal thing.